CONTRIBUTING.md

```diff
@@ -142,6 +142,16 @@ cd docs/_build/html/
 open _build/html/index.html
 ```
 
+## Write integration tests
+1. Create a .py file in correct subfolder of `tests/integrations` (ex: `test_wallet.py` in `tests/integrations/sugar`)
+2. Create a class that inherits `IntegrationTestCase` to store all individual tests under (ex: `class TestWallet(IntegrationTestCase)`)
+3. Create a function (can be sync or async) that contains logic for the test
+4. For customizability, include the `@test_async_and_sync` decorator on individual functions to test it on sync/async JsonRpc/Websocket clients, manage import modules, specify only websockets, allow a set number of retries, or use the testnet (vs standlone rippled node)
```

Comment:
This should explain what the decorator is doing and why it's important to have on almost all of the tests (and what types of tests it shouldn't be on)